### PR TITLE
Finished Translation ai-edu/A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 Preface.md

### PR DESCRIPTION
**Pull request by marko-polo-cheno**
Undo edit on Chinese version
Made new file for English version

To take careful note of:
Canadian vs American English spelling
Please revise (审查) the Notation and Convention section

### CODE REVIEW

File: A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 Preface.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+1. Judge which tasks are achievable by machine learning and which are science fiction. Don’t stay an amateur
+2. Thoroughly understand the basic theories of neural networks and deep learning
+3. Cultivate the ability to solve practical problems by analogies
+4. Have the ability to learn more complex models and advanced content alone
+5. For the more experienced readers, they can cultivate the ability to develop new models
+
+## Notation and Convention
+
+|Symbol|Meaning|
```
Comment (marko-polo-cheno) [inline A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 Preface.md:65]: Not confident in this section

File: A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md [hunk 2]
```diff
@@ -1,21 +1,23 @@
 <!--Copyright © Microsoft Corporation. All rights reserved.
   适用于[License](https://github.com/Microsoft/ai-edu/blob/master/LICENSE.md)版权许可-->
 
-# Preface
+# 前言
 
-**"What I cannot create, I do not understand." (American physicist Richard Feynman)**
+**What I cannot create, I do not understand.**
```
Comment (marko-polo-cheno) [inline A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md:6]: Changed Chinese file back to completely Chinese

File: A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 Preface.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<!--Copyright © Microsoft Corporation. All rights reserved.
+  适用于[License](https://github.com/Microsoft/ai-edu/blob/master/LICENSE.md)版权许可-->
+
+# Preface
+
+**"What I cannot create, I do not understand." (American physicist Richard Feynman)**
```
Comment (marko-polo-cheno) [inline A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 Preface.md:6]: New file for Enlgish version of 00 前言.md

**Comment by marko-polo-cheno** [comment]
I chose to finish this first, as I finished half of it already.

**Review by xiaowuhu — APPROVED** [review]

**Pull request merged**